manifests/02-rbac.yaml

```diff
@@ -95,6 +95,13 @@ rules:
   - create
   - get
   - update
+- apiGroups:
+    - config.openshift.io
+  resources:
+    - proxies
```

Comment:
this is in anticipation of the global proxy resource existing?

Reply:
Yes